userdocs/src/usage/managing-nodegroups.md

```diff
@@ -247,6 +247,8 @@ To ignore eviction rules such as PodDisruptionBudget settings, run:
 eksctl drain nodegroup --cluster=<clusterName> --name=<nodegroupName> --disable-eviction
 ```
 
+To speed up the drain process you can specify `--parallel <value>` for the number of nodes to drain in parallel.
+
```

Comment:
This is for only Unmanaged node groups, right?

Reply:
no, it works for both. I believe the original ticket was mistaken in trying to differentiate the two